hermes/bin/src/cli/package.rs

```diff
@@ -0,0 +1,30 @@
+//! Run cli commands module
+
+use std::path::PathBuf;
+
+use clap::Args;
+use console::Emoji;
+
+use crate::packaging::wasm_module::WasmModulePackage;
+
+/// Package cli command
+#[derive(Args)]
+pub(crate) struct PackageCommand {
+    /// Directory where placed all necessary files to package wasm module
+    /// A full description of the package can be found <https://input-output-hk.github.io/hermes/architecture/08_concepts/hermes_packaging_requirements/wasm_modules/#overview-of-a-wasm-component-module>
+    #[clap(long)]
+    dir: PathBuf,
+}
+
+impl PackageCommand {
+    /// Run cli command
+    pub(crate) fn exec(self) -> anyhow::Result<()> {
+        println!("{} Build wasm module package...", Emoji::new("📦", ""));
+
+        WasmModulePackage::from_dir(self.dir)?;
```

Comment:
Just throwing errors out the top of the application isn't acceptable.
It is not a good practice, and we need to properly handle them and report them.  Not just rely on a default "Print the error struct" catcher.

currenty this is doing this:
```
target/release/hermes package --dir /dosnt/exist
ℹ️
version: 0.0.1,
git info: {49e67f7308cf8d26d7d48b3fbb03423b12c10442.+ (feat/wasm-packaging)}
compiler: rustc 1.78.0 (9b00956e5 2024-04-29)
build time: 2024-05-22 04:06:02Z

📦 Build wasm module package...
🚨:
- H5Fcreate(): unable to create file: unable to open file: name = '/dosnt/exist/module.hdf5', errno = 2, error message = 'No such file or directory', flags = 13, o_flags = 242
```

We should be testing for the appropriate errors that can happen in normal use (like pointing to a directory that doesn't exist), and give a reasonable error message that explains why we failed and what they should do about it.

The error it currently gives is hostile and only a programmer would have any hope of understanding what it means.

Reply:
To be clear, "unexpected" errors can print a message like this, but they should still be caught and displayed in a nicer way than this.  This should only be for "unexpected" errors though, usually generated by a bug.  Errors that are possible due to bad parameters, or files are to be properly caught and handled.